### PR TITLE
adding new feature

### LINKED ISSUE #1: add changes to a file satimg
hi, 

make changes to requirement 

@JasneekD

### PR DESCRIPTION
**Pull request by JasneekD**
the pull request fixes #1 

@Annacummins

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,8 +1,10 @@
 GDAL==2.0.1
 numpy>=1.10,<1.11
 scipy==0.17.0
-scikit-learn==0.17
```
Comment (annacummins) [inline requirements.txt:4]: We do require this package - please don't remove this line

File: requirements.txt [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
+
 docopt==0.6.2
 matplotlib==1.5.1
 jupyter==1.0.0
```
Comment (annacummins) [inline requirements.txt]: We don't need this package - please remove this line

**Review by annacummins — CHANGES_REQUESTED** [review]
@JasneekD please review comments before approval

File: requirements.txt (hunk 2 above)
Comment (JasneekD) [reply requirements.txt]: @annacummins Thanks for your feedback and i made the changes :)

**Review by annacummins — APPROVED** [review]
Great - thank you

**Review by annacummins — DISMISSED** [review]
This is approved

File: requirements.txt [hunk 3]
```diff
@@ -6,3 +6,5 @@ docopt==0.6.2
...
```
Comment (annacummins) [inline requirements.txt]:
```suggestion
jupyter==1.0.1
```

**Review by annacummins — APPROVED** [review]

**Pull request merged**